crates/burrego/src/opa/default_host_callbacks.rs

```diff
@@ -0,0 +1,12 @@
+pub(crate) struct DefaultHostCallbacks;
+
+impl DefaultHostCallbacks {
+    pub(crate) fn opa_abort(msg: serde_json::Value) {
+        println!("OPA abort with message: {:?}", msg);
+        std::process::exit(1);
+    }
+
+    pub(crate) fn opa_println(msg: serde_json::Value) {
+        println!("Message coming from the policy: {:?}", msg);
```

Comment:
Do you think I should take care of doing `tracing` crate calls here?

Reply:
I think the default implementation should just use println for that stuff, not tracing. I would leave the code as it is right now.

Note well: the default implementation should be something designed for the burrego cli program.